docs/examples/swipingSlides.md

```diff
@@ -0,0 +1,23 @@
+## Swiping slides
+You can change the swiping behaviour with fastSwipe plugin, and you can disable swiping by setting props draggable={false}
```

Comment:
```suggestion
You can change the swiping behavior with the `fastSwipe` plugin, and you can disable swiping by setting props `draggable={false}`.
```

Reply:
and the same (`the` and the number of a given plugin in quotes) in other places though I'm not sure